evergreen/evergreen/urls.py

```diff
@@ -20,6 +20,15 @@
 from eg_app import views
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
-    path('',views.index,name='index'),
+    path("admin/", admin.site.urls), 
+    path('', views.indexHandler, name='index'), 
+    path('styles.css', views.fileHandler, name='styles'),  
+    path('functions.js', views.fileHandler, name='functions'), 
 ]
```

Comment:
We'll need `/register` and `/login` paths, too

Reply:
I have done this in feature/jslogin